pkg/controller.v1/common/service.go

```diff
@@ -28,10 +28,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/kubernetes/pkg/controller"
 )
 
+var (
+	succeededServiceCreationCount = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
Do we want to have a rough plan to discuss what kinds of metrics we want to bring to kubeflow/common?

Reply:
Yes I’ll put together a list.